bridge_ui/package.json

```diff
@@ -42,6 +42,7 @@
     "qrcode.react": "^3.0.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-i18next": "^14.1.2",
```

Comment:
This library takes care of using the right translation file based on the choice of the user (language switcher component will be implemented later, once we have some translations, for now we only have the English source file).